Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,312 @@
+        return res.json()
+
+    except Exception as e:
+        raise(e)
+
+
+def fetch_incidents():
+    now_utc = datetime.utcnow()
+    last_run_data = demisto.getLastRun()
+    last_run_time = last_run_data.get('time')
+
+    if last_run_time is None:
+        last_run_time = now_utc - timedelta(minutes=int(FETCH_INTERVAL))
+    else:
+        last_run_time = datetime.strptime(last_run_time, '%Y-%m-%dT%H:%M:%SZ')
+
+    all_issues = List_all_issues('false')
+
+    incidents = []
+
+    for issue in all_issues:
+        issue_create_time = issue['created_at']
+        issue_create_time = datetime.strptime(issue_create_time, "%Y-%m-%dT%H:%M:%SZ")
+
+        if issue_create_time > last_run_time:
+            incidents.append({
+                'name': str(issue['number']) + " - GitHub integration",
+                'occurred': issue['created_at'],
+                'rawJSON': json.dumps(issue)
+            })
+
+        if issue_create_time > last_run_time:
+            last_run_time = issue_create_time
+
+    demisto.incidents(incidents)
+    demisto.setLastRun({'time': last_run_time.isoformat().split('.')[0] + 'Z'})
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def get_issue_context(issue):
+    label_context = []
+    for l in issue['labels']:
+        label_context.append(l['name'])
+
+    assignee_context = []
+    for a in issue['assignees']:
+        assignee_context.append(a['login'])
+
+    context = {
+        'ID': issue['number'],
+        'title': issue['title'],
+        'state': issue['state'],
+        'locked': issue['locked'],
+        'body': issue['body'],
+        'assignees': assignee_context,
+        'labels': label_context
+    }
+    return context
+
+
+def List_all_issues(show_all):
+    suffix = '/repos/{}/{}/issues'.format(OWNER, REPO)
+    if show_all == 'false':
+        res = http_request('GET', suffix, None, None)
+    else:
+        res = http_request('GET', suffix, None, {'state': 'all'})  # include close issues
+
+    return res
+
+
+def List_all_issues_command():
+    show_all = demisto.args()['show-all']
+    issues = List_all_issues(show_all)
+    issue_list = []
+    for s in issues:
+        context = get_issue_context(s)
+        issue_list.insert(0, context)
+
+    header_list = ['ID', 'title', 'state', 'locked', 'body', 'assignees', 'labels']
+    human_readable = tableToMarkdown('issues list: ', issue_list, header_list)
+    entry_context = {
+        'Git.issue(val.ID && val.ID == obj.ID)': issue_list
+    }
+
+    return_outputs(human_readable, entry_context, issues)
+
+
+def create_issue(args):
+    suffix = '/repos/{}/{}/issues'.format(OWNER, REPO)
+    data = {}
+    for key in args.keys():
+        data[key] = args[key]
+
+    if 'labels' in data:
+        data['labels'] = data['labels'].split(',')
+
+    if 'assignees' in data:
+        data['assignees'] = data['assignees'].split(',')
+
+    data = json.dumps(data)
+    # return_error(data)
+    res = http_request('POST', suffix, data, None)
```

Comment:
you have a default value of None for the fourth element, so you don't have to send None to the function